pkg/apis/workspaces/v1alpha1/commands.go

```diff
@@ -1,16 +1,114 @@
 package v1alpha1
 
-type Command struct {
-	Actions    []CommandAction   `json:"actions,omitempty"`    // List of the actions of given command. Now the only one command must be specified in list; but there are plans to implement supporting multiple actions commands.
+import runtime "k8s.io/apimachinery/pkg/runtime"
+
+// CommandType describes the type of command.
+// Only one of the following command type may be specified.
+// +kubebuilder:validation:Enum=Exec;VscodeTask;VscodeLaunch;Custom
+type CommandType string
+
+const (
+	ExecCommandType         CommandType = "Exec"
+	VscodeTaskCommandType   CommandType = "VscodeTask"
```

Comment:
Also, it makes sense to consider renaming constant in upper case way, like `VSCODE_TASK_COMMAND` or `VSCODE_TASK_COMMAND_TYPE`
https://golang.org/doc/effective_go.html#constants